packages/react-components/react-button/package.json

```diff
@@ -40,6 +40,7 @@
     "@fluentui/react-tabster": "^9.6.5",
     "@fluentui/react-theme": "^9.1.7",
     "@fluentui/react-utilities": "^9.8.0",
+    "@fluentui/react-jsx-runtime": "9.0.0-alpha.1",
```

Comment:
_nit:_ Can you move this above `@fluentui/react-shared-contexts` in line `39` to keep alphabetical order?